.github/workflows/deploy-website.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
         with:
           persist-credentials: false
-      - uses: ruby/setup-ruby@v1.126.0
+      - uses: ruby/setup-ruby@c7079efafd956afb5d823e8999c2506e1053aefa # v1.126.0
```

Comment:
IIUC there are a bunch of [newer releases](https://github.com/ruby/setup-ruby/releases). Any idea why those aren't picked up?

Reply:
Nope, but I can take a look :).

Reply:
I see you figured something out: #706 :smile:

Reply:
Haha I wish, it did so by itself 😬, so I don't know why it didn't work before.

Reply:
Magic!